docs/tutorial.md

```diff
@@ -0,0 +1,93 @@
+
+To proceed with this guide I created a new Ethereum account - [0x0d4DF041Dbef6fFC0E444a4a213774AdB0c118C2](https://ropsten.etherscan.io/address/0x0d4DF041Dbef6fFC0E444a4a213774AdB0c118C2). I'm going to perform all the action on Ropsten Test Network, so you can follow. We encourage you first to use testnet untill a sufficient understanding of the CLN contracts is achieved.
+
+## Contract Addresses
+
+#### Ropsten Test Network
+- [ColuLocalNetwork](reference/ColuLocalNetwork.md) -  [0x41c9d91e96b933b74ae21bcbb617369cbe022530](https://ropsten.etherscan.io/address/0x41c9d91e96b933b74ae21bcbb617369cbe022530)
+- [ColuLocalNetworkSale](reference/ColuLocalNetworkSale.md) - [0xa973fa1cF412AC6A76C749aA6E1fcA7251814A48](https://ropsten.etherscan.io/address/0xa973fa1cF412AC6A76C749aA6E1fcA7251814A48)
+- [CurrencyFactory](reference/CurrencyFactory.md) - [0x7b2cbec58653aaf79842b80ed184b2ecb4e17d59](https://ropsten.etherscan.io/address/0x7b2cbec58653aaf79842b80ed184b2ecb4e17d59)
+- [IssuanceFactory](reference/IssuanceFactory.md) - [0xd352492ebeb9fad92147a3719766d7afe38fe26e](https://ropsten.etherscan.io/address/0xd352492ebeb9fad92147a3719766d7afe38fe26e)
+- [EllipseMarketMakerLib](reference/EllipseMarketMakerLib.md) - [0x30724fa809d40330eacab9c7ebcfb2a0058c381c](https://ropsten.etherscan.io/address/0x30724fa809d40330eacab9c7ebcfb2a0058c381c)
+
+
+#### Main Network
+
+- [ColuLocalNetwork](reference/ColuLocalNetwork.md) - [0x4162178B78D6985480A308B2190EE5517460406D](https://etherscan.io/address/0x4162178b78d6985480a308b2190ee5517460406d)
+- [ColuLocalNetworkSale](reference/ColuLocalNetworkSale.md) - [0x12321Fb3e2548b03eaD42F09cAb239cff377b4e2](https://etherscan.io/address/0x12321fb3e2548b03ead42f09cab239cff377b4e2)
+- [CurrencyFactory](reference/CurrencyFactory.md) - [0x21851f9970e333cbc253ba2c7ef953219c479ab7](https://etherscan.io/address/0x21851f9970e333cbc253ba2c7ef953219c479ab7)
+- [IssuanceFactory](reference/IssuanceFactory.md) - [0xe3444f1aac1a37b8cce47db260d1c6e25a2f627b](https://etherscan.io/address/0xe3444f1aac1a37b8cce47db260d1c6e25a2f627b)
+- [EllipseMarketMakerLib](reference/EllipseMarketMakerLib.md) - [0xc70636e0886eC4a4F2B7e42aC57ccD1B976352d0](https://etherscan.io/address/0xc70636e0886ec4a4f2b7e42ac57ccd1b976352d0)
+
+
+## Acquiring CLN
+
+For Mainnet, if you didn't get some tokens during the sale, you need to find someone to sell you some CLN. You can try [etherdelta](https://etherdelta.com/) or [forkdelta](https://forkdelta.github.io/), they are decentralize exchanges so we advice you to have a solid understanding of them before usage. Also, I suppose you already done this tutorial at least once if you intent to use it on Mainnet :smile:. So for Mainnet you can skip to the next section.
```

Comment:
https://ddex.io/trade/CLN-ETH 
is also dex for trading CLNs
after we will be listed in CMC we can give the market page of our cln listing here

Reply:
I think to add another little doc with a list of CLN listings and FAQ to buy